shared/settings/notifications/container.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React from 'react'
+import {connect} from 'react-redux'
+import Notifications from './index'
+
+import type {TypedState} from '../../constants/reducer'
+import type {Props} from './index'
+
+const NotificationsContainer = (props: Props) => (
+  <Notifications {...props} />
+)
+
+// TODO real integration
```

Comment:
this was what this branch was originally about, but i needed the settings nav working first (and its a blocker on other stuff). i'll branch off this branch to work on this next